templates/eks/components/eks-deployment-03.yml

```diff
@@ -145,7 +109,7 @@ Resources:
             sudo kubectl apply -f /curriki-eks/yamls/api.yaml
             sudo kubectl apply -f /curriki-eks/yamls/client.yaml
             curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.7.3 TARGET_ARCH=x86_64 sh -
-            escaped=$(echo "${AcmCertArn}" | sed 's%/%\\/%g')
+            escaped=$(echo ${AcmCertArn} | sed 's%/%\\/%g')
```

Comment:
Why removing quotes? Was this causing any issues?

Reply:
yes and to make it uniform